src/layout/Footer/Footer.tsx

```diff
@@ -1,9 +1,77 @@
-import { FooterWrapper } from './Footer.styled';
+import { AppBar, Button, Menu, MenuItem, Typography } from '@mui/material';
+import { StyledToolBar } from './Footer.styled';
+import LanguageRoundedIcon from '@mui/icons-material/LanguageRounded';
+import { useState } from 'react';
+
+const languages: string[] = ['English - en', 'Espanol - es'];
 
 export const Footer = () => {
+    const [selectedLanguage, setSelectedLanguage] = useState('English - en');
+    const [anchorElUser, setAnchorElUser] = useState<null | HTMLElement>(null);
```

Comment:
can we use some other vairable name and null will not cause any issues?

Reply:
okay, null will not cause any issues. Even in material UI docs also it is used like this.